### DISCUSSION TITLE
pelican-csi

### DISCUSSION THREAD

**Discussion by jjgraham** [PelicanPlatform/pelicanfs#66] [Ideas]
Hi All

Is there a plan for a pelican-csi ?

Can the CVMFS CSI be forked to be a pelican-csi
https://github.com/cvmfs-contrib/cvmfs-csi

Thanks

**Comment by turetske** [comment]
This is a repository for the PelicanFSSpec integration and issues created should be related to that. This isn't the forum for this discussion.

**Comment by turetske** [comment]
Converting this to a discussion so we can hopefully better address your needs and give more context.

**Comment by turetske** [comment]
So, to clarify. We have no plans to create a pelican-csi. Pelican isn't a file storage system, it's really more of an object transfer management system. The FSSpec is a bit of a misnomer in this case. Since the pelican-fs is based on the http fsspec, it's really doing the object access via the fsspec's http API rather than file management.
